conda.rb

```diff
@@ -66,10 +66,30 @@ def download_and_parse_packages
     channels_with_packages
   end
 
+  def latest(count)
+    packages = @redis.scan_each(match: "packages:*").to_a.map do |key|
```

Comment:
How many packages are there in total for conda? It may make sense to add a sorted set from the job also to avoid having to do the scan.

Reply:
1783